app/services/import_stories.rb

```diff
@@ -0,0 +1,35 @@
+class ImportStories
+  require 'rss'
+  require 'open-uri'
+
+  def call reset = false
```

Comment:
For consistency with the rest of the app, I'd use parenthesis around the args in the method definition

Reply:
Agreed!